docs/components/DocumentationFooter.tsx

```diff
@@ -1,10 +1,8 @@
-import styled, { keyframes, css } from 'react-emotion';
-
 import * as React from 'react';
-import * as Constants from '~/constants/theme';
+import { css } from 'react-emotion';
```

Comment:
Oh, this might cause some nice conflicts with the Emotion v10 upgrade

Reply:
Yep, this is why I'm waiting for that PR to land 😉

Reply:
It landed 😄